tensorflow/core/kernels/BUILD

```diff
@@ -995,7 +995,9 @@ tf_kernel_library(
         ":bounds_check",
         "//tensorflow/core:framework_lite",
         "//third_party/eigen3",
-    ],
+    ] + if_sycl([
```

Comment:
Instead of doing that can we add 
`if_sycl(["//tensorflow/core:sycl_runtime"])` to framework_lite ? ( https://github.com/lukeiwanski/tensorflow/blob/master/tensorflow/core/BUILD#L471 )